docs/src/main/asciidoc/native-reference.adoc

```diff
@@ -2235,3 +2235,24 @@ If the amount is not in hundreds, it could be a problem. A possible workaround i
 export JAVA_OPTS=-Djava.security.egd=/dev/urandom
 ----
 The proper solution is to increase the entropy available for the system. That is specific for each OS vendor and virtualization solution though.
+
+=== Work around missing CPU features
+
+When building on recent machines and running your native executable on older machines, you may see the following failure when starting the application:
+
+----
```

Comment:
This used to not look good on the website so let's make sure we have the `[source]` marker.
```suggestion
[source]
----
```